.storybook/components/StoryDetails/StoryDetails.css

```diff
@@ -16,5 +15,5 @@
   flex: 0;
   margin-top: var(--space-base);
   padding: var(--space-base);
-  background-color: var(--color-grey--lighter);
+  background-color: var(--color-surface--background--subtle);
```

Comment:
This was to fix the notice bar on Mobile Stories in Dark Mode:

Before:
![Screenshot 2024-11-21 at 2 00 45 PM](https://github.com/user-attachments/assets/1c256c3a-7bd6-4b56-9d0f-bc9035e0026a)


After:

![Screenshot 2024-11-21 at 2 01 03 PM](https://github.com/user-attachments/assets/095534ce-31da-455f-b7f8-655d559c752b)